admin/server-timing.php

```diff
@@ -70,13 +70,20 @@ static function() {
 					<br>
 					<?php
 					echo wp_kses(
-						sprintf(
-							/* translators: 1: Server-Timing, 2: template_include */
-							__( 'Since the %1$s header is sent before the template is loaded, only hooks before the %2$s filter can be measured.', 'performance-lab' ),
-							'<code>Server-Timing</code>',
-							'<code>template_include</code>'
+						str_replace(
+							'<a>',
+							'<a href="#server_timing_output_buffering">',
+							sprintf(
+								/* translators: 1: Server-Timing, 2: template_include */
+								__( 'Since the %1$s header is sent before the template is loaded, only hooks before the %2$s filter can be measured. Enable <a>Output Buffering</a> to measure hooks during template rendering.', 'performance-lab' ),
```

Comment:
I don't think we need the `str_replace()` here, that's a bit hacky. You can put the regular `<a href...` into the translation string, I'd recommend to make the actual `href` value a `%3$s` placeholder. That way we can get rid of the string replacement.

Reply:
Sure. I was thinking it felt more closely to [`createInterpolateElement`](https://make.wordpress.org/core/2020/07/17/introducing-createinterpolateelement/) but I'll change it to inline the `href` attribute.

Reply:
Done in 4868823